hazelcast/src/test/java/com/hazelcast/config/MapAttributeConfigReadOnlyTest.java

```diff
@@ -0,0 +1,19 @@
+package com.hazelcast.config;
+
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+
+public class MapAttributeConfigReadOnlyTest {
```

Comment:
haa, I just wanted to add it realising it myself and I see that both of you have already reviewed it ;)